tree/dataframe/src/RDataFrame.cxx

```diff
@@ -30,8 +30,8 @@ In a nutshell:
 ~~~{.cpp}
 ROOT::EnableImplicitMT(); // Tell ROOT you want to go parallel
 ROOT::RDataFrame d("myTree", "file_*.root"); // Interface to TTree and TChain
-auto myHisto = d.Histo1D("Branch_A"); // This happens in parallel!
-myHisto->Draw();
+auto myHisto = d.Histo1D("Branch_A"); // This books the (lazy) filling of a histogram
```

Comment:
What means lazy, the user may ask. So why not saying "This books the filling of the histogram, but does not yet run it."

Reply:
the line right below mentions the event loop is run there. i can take out the `(lazy)` here but I put it in brackets precisely so it can be ignored by users that don't understand it